app/gui/src/project-view/components/CircularMenu.vue

```diff
@@ -94,7 +96,19 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
             <SvgIcon name="paint_palette" class="rowIcon" />
             <span>Color Component</span>
           </MenuButton>
-          <MenuButton data-testid="edit-button" @click.stop="closeDropdown(), emit('startEditing')">
+          <MenuButton
+            v-if="isEnterable"
+            data-testid="enter-node-button"
+            @click.stop="closeDropdown(), emit('enterNode')"
+          >
+            <SvgIcon name="open" class="rowIcon" />
+            <span>Open Grouped Components</span>
+          </MenuButton>
+          <MenuButton
```

Comment:
Why do we remove edit node button if it is collapsed function? It can still be useful to edit the invocation?

Reply:
There are refinement notes, but those weren't updated to the newest design (with extended menu replacing full circular menu).

Reply:
Should the edit button not be removed in that case?

Reply:
Added it back.